cmd/transfer.go

```diff
@@ -99,7 +107,21 @@ func Transfer(ctx context.Context, rpcAddress string, chainID int64, mnemonic, i
 	if err != nil {
 		return err
 	}
+	if err := waitAndCheckStatus(ctx, chainA, tx); err != nil {
+		return err
+	}
 	log.Printf("3. sendTransfer success (TxHash: %s)\n", tx.Hash().Hex())
 
 	return nil
 }
+
+func waitAndCheckStatus(ctx context.Context, chain *geth.Chain, tx *types.Transaction) error {
+	receipt, err := chain.Client.WaitForReceiptAndGet(ctx, tx)
+	if err != nil {
+		return err
+	}
+	if receipt.Status != 1 {
```

Comment:
We should avoid using magic numbers. For here, you can use the following.
https://github.com/ethereum/go-ethereum/blob/6a6318b1d23ec20d1a8735cf1852c11726a4ac9a/core/types/receipt.go#L48

Reply:
https://github.com/datachainlab/ethereum-ics20-cli/pull/3/commits/8e980906f51af7880e39be53d99d0229444e05a0

Remove magic number, use const